functions/sanitiseGender.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/** Turns the user inputted gender into a tinyint
+ *
+ * @param string|null $gender user inputted gender
+ * @return int|null
+ */
+function sanitiseGender(?string $gender): ?int {
+    if ($gender === "female"){
+        return 1;
+    } elseif ($gender === "male") {
+        return 0;
+    }
+    return NULL;
```

Comment:
I don't LOVE the use of NULL here, it would be safer to use a third value of 2, but I'll allow it!

Reply:
Boolean NULL superiority is a contentious topic, when polled people were either for it, against it, or not available for comment